rush.json

```diff
@@ -488,6 +488,12 @@
       "reviewCategory": "libraries",
       "versionPolicyName": "rush"
     },
+    {
+      "packageName": "@rushstack/rush-sdk",
+      "projectFolder": "libraries/rush-sdk",
+      "reviewCategory": "libraries",
+      "versionPolicyName": "rush"
+    },
```

Comment:
Move this under the "libraries" category region in `rush.json`

Reply:
? It is already under "libraries"